app/views/static_pages/lessons.html.erb

```diff
@@ -1,16 +1,22 @@
 <% provide(:title, "Lessons") %>
-<div class="row justify-content-center">
-    <div class="col-md-6">
-      <h1>Categories</h1>
 
-      <div class="card-deck">
-        <div class="card">
-          <div class="card-body">
-            <h5 class="card-title"> Category </h5>
-            <p class="card-text">This is a longer card with supporting text below as a natural lead-in to additional content. This content is a little bit longer.</p>
+<h1>Categories</h1>
+
+<div class="card-deck">
+    <div class="card">
+        <div class="card-body">
+            <h5 class="card-title"> Caption </h5>
+            <p class="card-text">I'll do this in lesson answering function (bow)</p>
             <%= link_to "Start", class: "btn btn-primary" %>
-          </div>
         </div>
-      </div>
+
+        <div class="card">
+            <div class="card-body">
+                <h5 class="card-title"> Category </h5>
+                <p class="card-text">This is a longer card with supporting text below as a natural lead-in to additional
+                    content. This content is a little bit longer.</p>
+                <%= link_to "Start", class: "btn btn-primary" %>
+            </div>
```

Comment:
@shaniajubelag If I am not mistaken I think this file is not used anymore, so please delete